### PR TITLE
feat(card): Properly rearrange Card control templates and added elevated styles

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
﻿GitHub Issue: #14

## PR Type

What kind of change does this PR introduce?
- Feature

## Description
Properly rearrange Card control templates and added elevated styles

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- [ ] Tested MacOS
- [X] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Controls/Card.cs [hunk 1]
```diff
@@ -13,24 +14,44 @@ public partial class Card : Control
 		private const string __openedState = "Opened";
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Controls/Card.cs:14]: I know you didn't touch it, but can you change these two to $`{State}Name`?

File: src/library/Uno.Material/Styles/Controls/Card.xaml [hunk 2]
```diff
@@ -274,66 +300,317 @@
...
+								<Rectangle x:Name="ExpanderSeparator"
+										   Height="1"
+										   Margin="0,12,0,0"
+										   HorizontalAlignment="Stretch"
+										   Fill="{StaticResource OnSurfaceFocusedBrush}" />
+
+								<ContentPresenter x:Name="ExpanderPresenter"
+												  Content="{TemplateBinding ExpanderContent}"
+												  ContentTemplate="{TemplateBinding ExpanderContentTemplate}"
+												  Height="0" />
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Styles/Controls/Card.xaml:594]: intended?

File: src/library/Uno.Material/Controls/Card.cs [hunk 3]
```diff
@@ -13,24 +14,44 @@ public partial class Card : Control
...
 		private const string __closedState = "Closed";
 
-		#region Headear and SubHeader
-		public string Header
+		#region HeaderContent and HeaderContentTemplate
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Controls/Card.cs:17]: With this many properties we can probably move them into a separate file with `partial class`?

Comment (agneszitte-nventive) [reply src/library/Uno.Material/Controls/Card.cs:17]: Yes good idea, I will adjust that :) thanks for the feedback @XiaotianNetlift !

File: src/library/Uno.Material/Controls/Card.cs (hunk 1 above)
Comment (agneszitte-nventive) [reply src/library/Uno.Material/Controls/Card.cs:14]: I will make the change, thanks !

**Pull request merged**